packages/runtime/container-runtime/src/opLifecycle/opDecompressor.ts

```diff
@@ -30,9 +33,10 @@ interface IPackedContentsContents {
  */
 export class OpDecompressor {
 	private activeBatch = false;
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Too much fallout from changing?

Reply:
Yeah, unfortunately. I should leave a TODO here.
```suggestion
	// TODO: better typing
	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```